components/edit-collective/actions/Archive.js

```diff
@@ -49,7 +50,7 @@ const ArchiveCollective = ({ collective }) => {
   };
   const [archiveCollective] = useMutation(archiveCollectiveMutation, adminPanelMutationParams);
   const [unarchiveCollective] = useMutation(unarchiveCollectiveMutation, adminPanelMutationParams);
-  const isSelfHosted = collective.host?.id === collective.id;
+  const isSelfHosted = collective.host?.id === collective.id && collective.type !== CollectiveType.ORGANIZATION;
```

Comment:
```suggestion
  const isIndependentCollective = collective.host?.id === collective.id && collective.type == CollectiveType.COLLECTIVE;
```

Reply:
Can we please move forward there?

Reply:
Sorry, somehow seems I've missed this one. I've added the `isIndependentCollective` condition. But I don't see where we should use the `isHost` condition as we already have `collective.isHost`;

https://github.com/opencollective/opencollective-frontend/blob/29989675f9fd16748adad5bff28a70535a5507d7/components/edit-collective/actions/Archive.js#L153

So if the collective is not an independent collective and `collective.isHost` is true, then it should be a host (`isHost=true`).

Reply:
I looked at it again and proposed another simplification.